spec.bs

```diff
@@ -247,35 +260,55 @@ A user agent holds an <dfn>aggregatable report cache</dfn>, which is a [=list=]
 [=Implementation-defined=] values {#implementation-defined-values}
 ==================================================================
 
-<dfn>Max stored reports</dfn> is a positive integer that controls how many unsent reports can be queued waiting.
+<dfn>Maximum stored reports</dfn> is a positive integer that controls how many unsent reports can be queued waiting.
+
+<dfn>Maximum report contributions</dfn> is a positive integer that controls how many contributions can be present in a single report.
+
+<dfn>Minimum report delay</dfn> is a non-negative [=duration=] that controls the minimum delay to deliver an [=aggregatable report=]
 
-<dfn>Max report contributions</dfn> is a positive integer that controls how many contributions can be present in a single report.
+<dfn>Randomized report delay</dfn> is a positive [=duration=] that controls the random delay to deliver an [=aggregatable report=]. This delay is additional to the minimum delay.
 
 Issue: More
 
 Algorithms {#algorithms}
 ====================
 
+To <dfn>serialize an integer</dfn>, represent it as a [=string=] of the shortest possible decimal number.
 
-<!-- A {{WorkletGlobalScope}} |scope|'s <dfn algorithm for="PrivateAggregation">api identifier</dfn> is the result of:
-1. If |scope|'s [=worklet global scope type=] is {{FledgeWorkletGlobalScope}}, then return "<code>[=context type/fledge=]</code>".
-1. [=Assert=]: |scope|'s [=worklet global scope type=] is {{SharedStorageWorkletGlobalScope}}.
-1. Return "<code>[=context type/shared-storage=]</code>". -->
+Issue: This would ideally be replaced by a more descriptive algorithm in Infra. See infra/201
 
-To <dfn export algorithm for="PrivateAggregation">obtain a reporting endpoint</dfn> given an [=origin=] |reportingOrigin| and [=context type=] |api|, perform the following steps. They return a [=URL=].
+Scheduling reports {#scheduling-reports}
+----------------------------------------
+
+To perform the <dfn algorithm for="PrivateAggregation">report scheduling steps</dfn> with an [=environment settings object=] |context|, a [=context type=] |api| and a [=list=] of {{PAHistogramContribution}}s |contributions|:
```

Comment:
It seems a bit confusing that this algorithm is called "report scheduling steps" as it creates the report as well. Wdyt if we call it "report creation and scheduling steps?

Reply:
Sounds good, changed.